### ISSUE TITLE
ChaiScript 6.0 is old and does not build with a C++17 compiler

### ISSUE THREAD

**Issue by londey** [Microsoft/vcpkg#4109]
Machine: Windows 10 Pro 1803, Visual Studio Community 2017 15.8.1

# Repro
1. Set build setting `C++ Language Standard` to `ISO C++17 Standard (/std:c++17)`
1. `#include <chaiscript/chaiscript.hpp>` in a source file
1. Compile

# Expected
* Compiles without issue

# Actual
* ChaiScript fails to compile
```
vcpkg\installed\x64-windows\include\chaiscript\dispatchkit\callable_traits.hpp(99): error C2825: 'T': must be a class or namespace when followed by '::'
```

This issue issue seems to be addressed in ChaiScript 6.1

**Comment by MikeGitb** [comment]
Can this be closed now?

**Comment by londey** [comment]
@MikeGitb probably. The fix was merged and at this time 6.1 is still the latest version of ChaiScript
https://github.com/ChaiScript/ChaiScript/releases

**Comment by wangli28** [comment]
Hi @londey,
Latest version is 6.1, please open a new issue if this is a problem for you.
Thanks.
